Providers/Profile.swift

```diff
@@ -607,15 +627,17 @@ public class BrowserProfile: Profile {
         }
 
         private func endSyncing() {
+            // loop through status's and fill sync state
             syncLock.lock()
             defer { syncLock.unlock() }
             log.info("Ending all queued syncs.")
+            syncDisplayState = displayStateForEngineResults(syncReducer?.terminal.value)
```

Comment:
Instead of reaching for `syncReducer?.terminal.vaue`, since we call `endSyncing` afterwards like:
```swift
syncReducer.terminal >>> endSyncing
```
We can pass through the value:

```swift
syncReducer.terminal >>== endSyncing
...
private func endSyncing(results: [(EngineIdentifier, SyncResult)]) {}
```